h2o-py/h2o/model/model_base.py

```diff
@@ -511,11 +511,16 @@ def model_performance(self, test_data=None, train=False, valid=False, xval=False
                               data={"auc_type": auc_type})
             elif auuc_type is not None:
                 assert_is_type(auuc_type, Enum("AUTO", "qini", "gain", "lift"))
+                assert_is_type(custom_auuc_thresholds, [float], None)
```

Comment:
These changes enable custom thresholds to calculate metrics and could be in a different PR, but it is necessary for tests. But I can move it.